update Quantinuum Target Availability descriptions (#560)
MicrosoftDocs/quantum-docs: articles/provider-quantinuum.md

Technical details for the System Model H1 and System Model H1 Emulators can be found in Quantinuum's [product data sheets](https://www.quantinuum.com/products/h1).

## Target Availability

> The Quantinuum H-Series quantum computers were designed to be continuously upgraded, which allows customers to have access to the latest and greatest hardware capabilities as our teams continually improve gate fidelities, memory errors, and system speed. 

Reviewer: ```suggestion
The Quantinuum H-Series quantum computers are designed to be continuously upgraded, which allows customers to have access to the latest hardware capabilities as Quantinuum continually improves gate fidelities, memory errors, and system speed. 
```

> The Quantinuum hardware cycles through commercial and development cycles. During commercial weeks, hardware is available to process jobs via a queue system. During development weeks, the hardware is offline as upgrades are completed.

Reviewer: ```suggestion
Quantinuum hardware cycles through commercial periods and development periods. During commercial periods, the hardware is available to process jobs via a queue system. During development periods, the hardware is offline as upgrades are applied.
```

> A calendar is sent to Quantinuum users monthly with information on the commercial and development cycles. If you have not received this please email QCsupport@quantinuum.com.

Reviewer: ```suggestion
Every month, a calendar is sent to Quantinuum users with information on the commercial and development periods. If you have not received this calendar, please email QCsupport@quantinuum.com.
```

A target's status indicates its current ability to process jobs. The possible states of a target include:

- **Available**: The target is online, processing submitted jobs and accepting new ones.
- **Degraded**: The target is accepting jobs, but not currently processing them.
- **Unavailable**: The target is offline, not accepting new job submissions.

> For the Quantinuum quantum computer targets, **Available** and **Degraded** correspond to commercial cycles, while **Unavailable** corresponds to development cycles while the machine is offline for upgrades.

Reviewer: ```suggestion
For the Quantinuum quantum computer targets, **Available** and **Degraded** correspond to commercial periods, while **Unavailable** corresponds to development periods where the machine is offline for upgrades.
```
